fontforge/ufo.c

```diff
@@ -107,6 +107,16 @@ static void injectNumericVersion(char ** textVersion, int versionMajor, int vers
   return;
 }
 
+/* The spec does not really require padding the version str but it clears */
+/* 1.8 vs. 1.008 ambiguities and makes us inline with the AFDKO. */
+char* paddedVersionStr(const char * textVersion, char * buffer) {
+    int major = -1;
+    int minor = -1;
+    if (!extractNumericVersion(textVersion, &major, &minor)) return (char *) textVersion;
+    snprintf(buffer, 6, "%d.%03d", major, minor);
+    return buffer;
+}
+
```

Comment:
I cannot confirm or dispute that it is desirable to add zeros like that. So I will call upon @DavidRaymond and @miguelsousa.